### ISSUE TITLE
Etrian Odyssey III save problems

### ISSUE THREAD

**Issue by diuXd9gfdwco2P** [ahezard/nds-bootstrap#396]
Hello,
My EOIII game just decided to delete my save yesterday after 20 hours of gameplay.

Since it's not a known issue anywhere with this game, I decided to post it here.

The game always had save issues for me. Whenever you save, power off/on the console and reload, there is always some map data which gets erased. Only the map data though. My characters, quests and overall progression remains untouched. Until yesterday at least when I powered off my console by reflex, it then decided to go all in for some reason.

When you load the game without powering off the console (e.g when you lose a battle), it loads correctly.

I don't know if it is supposed to be normal, but my save file is still on my SD card. Here it is if you need it : https://ufile.io/i6p0p . If you can tell me at the same time if my save is really lost or not, I'd be very, very grateful.

Thanks.

**Comment by Kalanyr** [comment]
I had a look at your sav file with a hex editor and there's definitely still data of some kind in it (and it's not completely random as the header is still there and there seems to be the kind of patterns in it you'd expect from something storing eg map progress).  I'd suggest trying it in a DS emulator of some kind and seeing if it works in that, and if it doesn't it might be worth seeing if anyone has written out the EO3 save format and checking if it's munged beyond repair or if it's salvageable.

**Comment by diuXd9gfdwco2P** [comment]
Hey,
Thank you very much for taking your time to look at the save file. I already tried loading the save in Desmume but the game still doesn't recognize the save.  I tried the same thing with other DS games just in case, and they load correctly.

I'll check other emulators and tinker with some of the options to see if I can get something done.

Nonetheless, I'm still curious to know what caused the data loss and if someone else using nds-bootstrap has experienced the same problems.
